Modules/_blake2/blake2b_impl.c

```diff
@@ -59,9 +61,11 @@ new_BLAKE2bObject(PyTypeObject *type)
 {
     BLAKE2bObject *self;
     self = (BLAKE2bObject *)type->tp_alloc(type, 0);
-    if (self != NULL) {
-        self->lock = NULL;
+    if (self == NULL) {
+        return NULL;
     }
+    INIT_MUTEX(self);
```

Comment:
Maybe `HASHLIB_INIT_MUTEX` to make it a bit more clear where the macro is coming from.